src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/GoogleDataprocDAO.scala

```diff
@@ -1,22 +1,26 @@
 package org.broadinstitute.dsde.workbench.leonardo.dao
 
-import com.google.api.services.dataproc.model._
-import org.broadinstitute.dsde.workbench.leonardo.config.DataprocConfig
-import org.broadinstitute.dsde.workbench.google.GoogleUtilities
 import akka.actor.ActorSystem
 import akka.http.scaladsl.model.StatusCodes
+import cats.data.OptionT
+import cats.instances.future._
 import com.google.api.client.auth.oauth2.Credential
 import com.google.api.client.googleapis.auth.oauth2.GoogleCredential
 import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport
 import com.google.api.client.googleapis.json.GoogleJsonResponseException
 import com.google.api.client.json.jackson2.JacksonFactory
+import com.google.api.services.compute.Compute
+import com.google.api.services.compute.model.Instance
 import com.google.api.services.dataproc.Dataproc
+import com.google.api.services.dataproc.model.{ClusterStatus => _, _}
 import com.google.api.services.pubsub.PubsubScopes
-import org.broadinstitute.dsde.workbench.leonardo.model.{ClusterRequest, ClusterResponse, LeoException}
-import org.broadinstitute.dsde.workbench.model.{ErrorReport, WorkbenchExceptionWithErrorReport}
+import org.broadinstitute.dsde.workbench.google.GoogleUtilities
+import org.broadinstitute.dsde.workbench.leonardo.config.DataprocConfig
 import org.broadinstitute.dsde.workbench.leonardo.errorReportSource
+import org.broadinstitute.dsde.workbench.leonardo.model.ClusterStatus.ClusterStatus
 import org.broadinstitute.dsde.workbench.leonardo.model.ModelTypes.GoogleProject
-
+import org.broadinstitute.dsde.workbench.leonardo.model.{Cluster => _, _}
```

Comment:
can we be super-explicit and rewrite leo's `Cluster` to `LeoCluster`

Reply:
or even name it as such in the model.  Maybe `ManagedCluster` ?

Reply:
`ManagedCluster` in the model sounds good to me..

Reply:
..however I'll hold off for now, I think that kind of refactoring would be better in a separate PR.

Reply:
ok